doc: charge-rpg (party-extra) (#31)
fariapp/fari-games: public/documents/fari-rpgs/charge-rpg.md

\- By David Haslem

## Overview

The group of players is referred to in this extra as a `party`. This group has a separate `party sheet` which tracks `details` of how they are perceived by the world around them, as well as maintaining party-level `danger` and `progress` clocks.

The `party sheet` can represent things such as their “HQ” or “Ship”. It can also be used to represent relationships with important groups or specific NPCs. 

> #### Example
> 
> The play group wants to tell the story of a small band of resistance fighters against an evil space empire. They decide they want to start their story with a stolen prototype spaceship and not much else. They have some ideas for characters and they know they want to start on an out-of-the-way space station.

###  Key Concepts and Supporting Details

The `party sheet` tracks four `key concepts`, representing how the party is positioned along four axes. Each `key concept` may have as many as four `supporting details`, which have a `disposition` of positive, negative, or neutral. The combination of these `supporting details` contributes to the overall `key concept` being positive, negative, or neutral.

> When `key concepts` or `supporting details` are positive they grant improvements to related rolls the party makes. Negative `key concepts` or `supporting details` cause worse `position` on related rolls.

Reviewer: question: is only the key concept evaluated when making action rolls ? or can supporting details evaluated on their own?

The phrasing looks a bit weird to me, but I'm having a hard time coming up with something better. Maybe we can ask the server for feedback.

Reviewer 2: I also have some difficulty to understand this part.
I think it'd better to explain the disposition first, then the key concept and the supporting details.

Also, what kind of improvements are granted?

Author: My thinking is that `key concepts` deal with broad categories and `supporting details` are very specific. The difference between "+1 effect when dealing with this particular NPC" vs "+1 effect dealing with any friendly NPC". 

Benefits being either position or effect. I'll see how I can rewrite this to communicate that a bit better.

Author: Maybe this version better communicates my intent? 

```
The `party sheet` tracks `supporting details` about the party. These details have a `disposition` which might be positive, negative, or neutral. The `disposition` impacts either the `position` or `effect` of rolls related with the specific detail on the `party sheet`.

`Supporting details` are collected into four categories, summarized with four broad `key concepts`. If the majority of details under the `key concept` are positive (or negative), the `disposition` of the `key concept` also becomes positive (or negative). The `disposition` of the `key concept` works the same as `supporting details`, but at a broader scope, impacting all rolls related to the category.
```

Reviewer 2: Perfect, the new version is way better for me.

>
> The party decides on the following to start their `party sheet` to represent the story they came up with:
>
> Authority of -2, representing the evil empire, with a `key concept` of “Wanted by the Kabraxian Empire”, and two negative `supporting details` of “thorn in the side of the regional governor” and “falsified spacecraft registration”  
>
> Assets of +1, with a `key concept` of “scrappy band of fighters ready to fight” and positive `supporting detail` of “recently ‘liberated’ state-of-the-art spacecraft”.
>
> Allies of +1, with a `key concept` of “nobody likes the Empire”, a positive `supporting detail` of “Maven, a bar owner friendly to rebels” and a neutral `supporting detail` of “dock workers tired of Empire soldiers pushing them around”.
>
> Adversaries of 0, with a `key concept` of “enemies of my enemies are my friends?” and a single neutral `supporting detail` of “Jebralt, the sketchy arms dealer”.
>
> All of these `key concepts` are still neutral in `disposition`, but Authority is close to becoming negative. The team will have to keep a low profile for a while if they want to avoid being put into “desperate” positions with the Empire before they’re ready. 

> 

Reviewer 2: Switching from narrative supporting details and disposition to numerical value is a bit weird and confusing.
Maybe earlier in the disposition description's it would be great to add a sentence about how to convert from narrative to value, just to let the reader be warned about this before discovering it in the example.

Author: Yeah, I'm going to remove the numbers completely from the example - I agree it is a bit confusing.

> 

Author: Just a fun stock image or are you thinking a shot of a "party sheet" to help flesh out the example?

### Party Danger and Progress

When it makes sense as a `consequence` of events in the session, the GM may mark the `danger` clock on the `party sheet`. This clock tracks a generally deteriorating position in society - loss of assets, adversaries growing more hostile, or alliances shifting. In a similar fashion, the `progress` clock is available for when significant events happen in a session that might improve the party’s position. 

> When these clocks fill, they are reset and a `supporting detail` changes. If the `danger` clock fills, this change is generally for the worse. If the `progress` clock fills, the change is generally for the better. 

Reviewer: ```suggestion
When any of these clocks are filled, they are cleared and one of the `supporting detail` changes. If the `danger` clock fills, this change is generally for the worse. If the `progress` clock fills, the change is generally for the better. 
```

### Burning Progress

Sometimes a `party` will have to sacrifice their long term goals to focus on the short term goals. If it fits the narrative, `progress` can be burned as `momentum` to represent this on a one-for-one basis.

## Concepts and Details

> #### Designer Note
> 

> > Key Concepts and Supporting details can be a way for game designers to help drive the kinds of stories that make sense in their setting. The categories provided below are intended to be a generic starting point that can be made setting specific - renaming them per setting is an easy customization to set the right mood!

Reviewer 2: ```suggestion
> `Key Concepts` and `Supporting details` can be a way for game designers to help drive the kinds of stories that make sense in their setting. The categories provided below are intended to be a generic starting point that can be made setting specific - renaming them per setting is an easy customization to set the right mood!
```

> 
> Further structure can be added to the supporting details to drive home key points in the setting, for example, asking players to choose a specific adversary and ally from a list of available world forces.
> 
> Starting positions for these key details can help further drive a story for the group of players to build on. A crew looking for their first big score might not have a lot of assets or allies, but they probably aren’t big enough to be on anyone’s radar as an adversary either. A resistance group against a repressive regime might start in a horrible position against authority, but have a larger list of allies to lean on. 

> Like with character creation, `details` on the `party sheet` can offer both fictional support (helping build on the story) and narrative liberties (giving players permission to take certain actions). While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. There will also have the chance to change these over time as the `party` advances.  

Reviewer: question: is advances a term ? I understand we probably dont want to use "progress" but I havent seen the term "advance" used often

> Like with character creation, `details` on the `party sheet` can offer both fictional support (helping build on the story) and narrative liberties (giving players permission to take certain actions). While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. There will also have the chance to change these over time as the `party` advances.  

Reviewer 2: I'm on the same page, I think both (...) aren't necessary. If you want to emphasize about players permissions, maybe "players narrative opportunities" ca fit?

For "advances", I have the same feeling as RP. Maybe "create its own story"?

> Like with character creation, `details` on the `party sheet` can offer both fictional support (helping build on the story) and narrative liberties (giving players permission to take certain actions). While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. There will also have the chance to change these over time as the `party` advances.  

Author: Agreed - this wording is awkward. Just trying to communicate that these details are the same as any others - even neutral ones can grant narrative liberties, etc. Let me see if I can make it a bit clearer:

```suggestion
Like with character creation, `details` on the `party sheet` are facts about the party that can be used for story telling. They can also offer narrative liberties - potentially giving players permission to take certain actions. While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. Much like character details, there will also have the chance to change these over time through play.```

> Like with character creation, `details` on the `party sheet` can offer both fictional support (helping build on the story) and narrative liberties (giving players permission to take certain actions). While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. There will also have the chance to change these over time as the `party` advances.  

Reviewer 2: " They also offer narrative liberties, potentially giving..."
The "can" is imo too much here, and a comma sounds better.
(sorry I can't insert a diff from the suggested change)

### Key Concepts

Each `key concept` focuses on the position of the `party` along an important axis. The `disposition` of the party in these categories may be represented as the sum of positive `supporting details`, subtracting any negative `supporting details`, resulting in a range from -4 to +4. 

> `Supporting details` are specific people, places, things or ideas related to the party in some sort of way, while `key concepts` are overarching, category level descriptions of the party’s standing. 

Reviewer: question: are supporting details like World Forces? Should those two terms be tied in somehow?

Author: Yeah, I think I'm going to reword this a bit and try to make it a bit more compare and contrast the two as well.

> * **Assets** - This represents the available resources of the party, including money and any available equipment.

Reviewer: suggestion: perhaps we should say something like "There are 4 `key concepts`:" and then list them?

Author: Agree

* **Authority** - This represents the party’s relationship with authority - how much they have, or how much those who have it like (or dislike) the party.

Examples for each `key concept` and related `supporting details` are available further on in this document.

### Dispositions 

`Supporting details` have either a positive disposition (good for players, +1 to key concept), or a neutral disposition (+0 to key concept), or a negative disposition (bad for players, -1 to key concept). These dispositions can be used to alter the `position` of a roll. `Disposition` can alter the consequence rating of a `position`, and might be directly used by the GM to help set the `position` of a roll when it makes sense.

| Disposition                  | Default Position                 | Consequence Rating               | 
| ---------------------------- | -------------------------------- | -------------------------------- |
| Positive (+)                 | Controlled                       | -1 Consequence Rating            |
| Neutral  (0)                 | Risky                            | No change to Consequence Rating  |

> | Negative (-)                 | Desperate                        | +1 Consequence Rating            |

Reviewer: nice: cool to see a table. I feel it helps a lot with understanding this extra

> The collection of `supporting details` for a `key concept` ladder up to a numeric score between -4 and +4. A `key concept` is considered neutral until the score reaches either <-2 (three or more negative `supporting details`), becoming a negative `disposition`, or >+2 (three or more positive `supporting details`), becoming a positive `disposition`. When the `key concept` changes disposition it should be rewritten to represent the change in fortunes for the party.

Reviewer: question: why does it take -2 or +2 to become positive or negative? I feel like it adds complexity on how to calculate the rating of a key detail.

Author: I'm going to try to change the wording above to better explain it, and rather than using the number, say disposition of the "key concept" matches the majority of "supporting details"

```markdown
### Zooming in and out

Each `key concept` is a `detail` that focuses broadly on the position of the 
`party` in an important category. Narratively they zoom out and share more 
of the big picture. `Key concepts` are wide-reaching - they create an assumed 
default for anything related to that category that isn't already a `supporting 
detail`.

`Supporting details` add color to `key concepts` by zooming in on specific people, 
places, or things related to the party in some way (for example, a relationship with 
a specific `world force`).
```

**Negative (-) Supporting Details**

Negative `supporting details` for assets might be a big loan that the party can never quite pay off, or a broken down ship that will cost more than it’s worth to repair.  

#### 2. Authority

This represents how people in power feel about the party, or, depending on the game how much power the party has. A positive `key concept` might be something like “leaders of a thriving business empire”, while a negative one might be something like “considered enemies of the kingdom”. 

**Positive (+) Supporting Details**

Positive `supporting details` for authority might include a group of freedom fighters ready to follow your lead or a being the king’s go-to problem solvers. 

**Neutral (0) Supporting Details**

Neutral `supporting details` for authority might be an easily bribed bureaucrat willing to look the other way for now, or a legal decree that could end up causing trouble for the party. 

**Negative (-) Supporting Details**

Negative `supporting details` for authority might be the captain of the watch out to get you or a bounty on everyone’s heads for their latest crime. 

#### 3. Allies

> This represents people that might be willing to help the party, or who have reason to be connected with the party in some way. While typically positive sounding, negative `dispositions` could be used to represent obligations or relationships turned sour. A positive `key concept` might be something like “friends to all we meet”, while a negative one might be something like “everyone expects our help whether they deserve it or not”. 

Reviewer: question: "friends to all we meet" first time I see those words being laid out in this order. Is that grammatically correct?

Author: changing to more obvious - "lots of powerful allies"

**Positive (+) Supporting Details**

Positive `supporting details` for adversaries could be people or groups that might still help the party if they were in mortal danger, sworn enemies that are comically bad at their jobs, or maybe enemies that don’t yet realize the party exists.

**Neutral (0) Supporting Details**

Neutral `supporting details` for adversaries could be enemies that have bigger fish to fry and consider the party beneath them, or powerful desert sandstorms that can be used to the party’s advantage if they can survive. 

**Negative (-) Supporting Details**

Negative `supporting details` for adversaries might include powerful enemies sworn to destroy the party, or impending doom of an inbound asteroid.  

## Danger and Progress

> #### Designer Notes
> 
> The pace at which the `supporting details` of the party change can be a dial for changing the feel of the game, especially in games where additional mechanical benefits are offered for building up positive `supporting details`.  By default, the intended pace is an average of no more than one detail changing per session.

> The `party sheet` holds two clocks which fight against each other, one dragging the party down, and the other pulling the party up. By default, both of these clocks have eight segments. 

Reviewer: suggestion: OK here's the information I wanted above. I feel like the section above should include this or perhaps both sections should be mixed up somehow ? What do you think?

### Taking Consequences

> The GM or players may suggest that marking a consequence on the `party sheet` makes sense. Generally this should represent a major consequence as it can have long-term implications, and should not be taken lightly - a couple of times per session is probably a good rule of thumb. This adds segments to the danger clock of the party sheet.

Reviewer: > The GM or players may suggest that marking a consequence on the `party sheet` makes sense
OK, when should this happen? I have the impression the information in this paragraphe should be switched up like.

Like:
- There will be times when something very bad happens to this party
- When this happens, the group can agree to tick a danger clock on the party sheet (I see you are talking about "marking a consequence" but I don't think its a thing??)
- This should usually not happen more than once per session.

Add a new `supporting detail` - the `disposition` of this detail should generally start at `neutral`. Note `supporting details` should all be specific, unique details that tie back into the fiction that are expected to show up again. 
Downgrade the `disposition` of an existing `supporting detail` (moving from `positive` to `neutral` to ` negative`).
Remove or replace an existing neutral `supporting detail`. If all the spaces that make sense are taken or negative, consider whether recent events might have completely supplanted an existing `supporting detail`. This could be a good fit for story elements that have served their purpose or didn't really get much play. 
Create a new immediate threat - the GM exposes a danger and threatens the players with a negative `supporting detail` _unless_ they complete a mission to prevent it.

> #### Example
> 
> The danger clock fills from the strain placed on the starship engines racing to help a dying friend.
> 
> The party decides it makes the most sense to downgrade the Assets `supporting detail` “state-of-the-art ship” (+) to “damaged ship” (0). 

### Marking Progress

> Generally players should be given a chance to advance the progress clock a couple of times per session - this can be as the result of completing a project and/or as a reward for successful completion of a mission. 

Reviewer: suggestion: Marking Progress VS Taking Consequence. The verbiage is different but those are two interwined concepts I believe. I feel like its like "Ticking the PArty Progress Clocks" and "Ticking the Party Danger Clocks"

> #### Example
> 
> Saving a person in need has completed a story arc for the players. The GM grants them a milestone and two progress on the `party sheet`. One player has a long term project to find allies and rolls successfully during downtime, adding two more. 

### Burning Progress

> The segments of the progress clock can be used as a shared pool of momentum. Players may decide to sacrifice their long term goals for a short term boost and spend filled segments of the progress clock as momentum.

Reviewer: question: this was already touched on in the section above. Is that duplicated information?

### Filling the Progress Clock

When the progress clock fills up the players should decide, based on recent events, what makes the most sense in the storyline:
Add a new `supporting detail` - the `disposition` of this detail should generally start at `neutral`. Note `supporting details` should all be specific, unique details that tie back into the fiction, so this is a good chance to promote NPCs to recurring characters, or lean into interesting bits of the story.
Upgrade the `disposition` of an existing `supporting detail` (moving from `negative` to `neutral` to `positive`).
Remove or replace an existing neutral `supporting detail`. If all the spaces that make sense are taken or positive, consider whether recent events might have completely supplanted an existing `supporting detail`. This could be a good fit for story elements that have served their purpose or didn't really get much play.

> Create a new immediate opportunity - the GM exposes an opportunity and offers the players with a positive `supporting detail` _if_ they complete a mission to obtain it.

Reviewer: must: this should be a list

## Other Uses for Key Concepts and Supporting Details

> #### Designer Note
> 
> Certain games may pick and choose between the following options - while they do not add a large amount of complexity, using the full set of rules grants the players many additional options to see them through tough spots. 

### Supporting Details Grant Conditional Position

When dealing with a situation that directly relates to a `supporting detail`, the `disposition` of the supporting detail (positive or negative) may be used to help determine the `position` of the roll. 

### Key Concepts Grant Conditional Position

When dealing with a situation that might benefit from a positive `key concept`, the `disposition` of the `key concept` may be used to help determine the `position` of the roll. Since `key concepts` are generic to category descriptors, this gives a much broader benefit than individual `supporting details`. 

> The same works in reverse - allowing a `key concept` to become broadly negative can make the lives of the party more desperate.

Reviewer: suggestion: I thinks those two things could be wrapped inside the same category like "How the party sheet affects the fiction" or something (bad example but you get the idea 😅 )

### Improved Effect

When dealing with a situation for which a `supporting detail` or `key concept` could be a useful advantage, players may opt for +1 effect rather than an improved position. Which option is being used must be decided before the roll. 

### Losing Progress via Add Tension

> Progress toward improving details can be spent as momentum. Taking it a step further - if it makes sense, the players might suggest downgrading the `disposition` of a supporting detail as a consequence to `add tension` if they don’t have progress to spare. 

Reviewer: nice: I love this

## Dials

These rules can be adapted in a few different ways. Below are some ways to tweak them to better fit how the play group wants to use them. 

### No Clocks

> Rather than track progress and danger on abstract clocks, the play group might decide to update the `party sheet` whenever it makes sense narratively. Rather than keep track of additional clocks, the players can decide to update details 

Reviewer 2: ```suggestion
Rather than track progress and danger on abstract clocks, the party might decide to update the `party sheet` whenever it makes sense narratively. Rather than keep track of additional clocks, the players can decide to update details 
```

You alrady define the term, and the repetition here can serve as an emphasis to the extra
